kerykeion/utilities/core.py

```
@@ -326,6 +326,97 @@ def is_point_between(
+        The twelve widths, and for each the flag saying it was measured against
+        the direction of the frame it was given.
+    """
+    forward = [normalize_degree(cusps[(index + 1) % 12] - cusps[index]) for index in range(12)]
+    if abs(sum(forward) - 360.0) <= _HOUSE_WINDING_TOLERANCE_DEGREES:
+        return forward, [False] * 12
+
+    backward = [normalize_degree(cusps[index] - cusps[(index + 1) % 12]) for index in range(12)]
+    if abs(sum(backward) - 360.0) <= _HOUSE_WINDING_TOLERANCE_DEGREES:
+        return backward, [True] * 12
+
+    shorter = [ahead <= behind for ahead, behind in zip(forward, backward)]
```

Ruff (0.16.2)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)